src/main/java/frc/robot/Robot.java

```diff
@@ -66,8 +68,7 @@ public void robotInit() {
     arm = new Arm(armGroup, RobotMap.ARM_ENCODER_A_CHANNEL, RobotMap.ARM_ENCODER_B_CHANNEL, RobotMap.ARM_ENCODER_PPR,
         RobotMap.ARM_GEAR_RATIO);
 
-    wrist = new Wrist(RobotMap.WRIST_MOTOR_CHANNEL, RobotMap.WRIST_ENCODER_A_CHANNEL, RobotMap.WRIST_ENCODER_B_CHANNEL,
-        RobotMap.WRIST_ENCODER_PPR, RobotMap.WRIST_GEAR_RATIO);
+    wrist = new Wrist(RobotMap.WRIST_CAN_ID, MotorType.kBrushless, RobotMap.WRIST_GEAR_RATIO);
```

Comment:
I would not pass all the RobotMap constants into a constructor. Those constants are meant to be global constants, so in theory, there should be no reason to pass them into a constructor as it should be available within the Wrist class. You typically only want to pass in objects that you can't get to within your Wrist class without passing them in. These will usually be other objects that you have created in this case the Robot class that the Wrist relies on.

Reply:
My goal with writing it this way was to make it so that if we changed the robot physically we would only need to change what's in the constructor rather than the class itself. I try to generalize classes as much as I can, then make them more specific when I implement them. Basically, once I'm finished writing a class, I want to worry about what implements the class rather than the class itself.